_data/tests.yml

```diff
@@ -4,26 +4,36 @@ tests:
     - name: "4.4"
       project_url: "https://qa-reports.linaro.org/api/projects/40/"
       squad_url: https://qa-reports.linaro.org/lkft/linux-stable-rc-4.4-oe/
+      order_num: 1
```

Comment:
Please implement this without 'order_num'; a list is already an ordered data structure.